.github/workflows/pr-to-nostr.yaml

```diff
@@ -0,0 +1,29 @@
+name: Nostr PR Notifier
+on: 
+  pull_request:
+    types: [opened, reopened]
+jobs:
+  Notify:
+    runs-on: ubuntu-latest
+    env:
+      # Set in Github repo > Settings > Secrets and Variables
+      NOSTR_KEY: ${{ secrets.NOSTR_KEY }}
+      # Event vars
+      PR_URL: ${{ github.event.pull_request.html_url }}
+      PR_TITLE: ${{ github.event.pull_request.title }}
+      PR_AUTHOR: ${{ github.event.pull_request.user.login }}
+      PR_BODY: ${{ github.event.pull_request.body }}
+    steps:
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.19.0'
+      - run: go install github.com/fiatjaf/noscl@latest
+      - run: which noscl
+      - run: noscl setprivate $NOSTR_KEY
+      - run: noscl relay add wss://relay.damus.io
```

Comment:
Probably want to move these into a variable, but then you'll need to split on a comma/etc. Figured I'd land this first and we can go from there.